packages/mermaid/src/docs/intro/syntax-reference.md

```diff
@@ -8,7 +8,13 @@ Diagram Examples can be found in the [Mermaid Live Editor](https://mermaid.live)
 
 ## Syntax Structure
 
-One would notice that all **Diagrams definitions begin** with a declaration of the **diagram type**, followed by the definitions of the diagram and its contents. This declaration notifies the parser which kind of diagram the code is supposed to generate.
+One would notice that all **Diagrams definitions begin** with a declaration of the **diagram type**, followed by the definitions of the diagram and its contents. This declaration notifies the parser which kind of diagram the code is supposed to generate. The only exception to this a [Frontmatter](#frontmatter-for-diagram-code) configuration.
+
+Code is case insensitive. Indents are completely ignored but recommended for structure and readability.
```

Comment:
```suggestion
```

This is not a general statement we can make. There are case & indentation sensitive diagrams.